zips/zip-0317.rst

```diff
@@ -310,7 +325,7 @@ calculated in zatoshis per the following formula:
      zsa\_logical\_actions  &=& logical\_actions \;+ \\
                        & & nTotalOutputsZsaIssuance \\
      zsa\_conventional\_fee &=& marginal\_fee \cdot \mathsf{max}(grace\_actions, zsa\_logical\_actions) \;+ \\
-                       & & issuance\_fee \cdot nIssueActions \;+ \\
+                       & & issuance\_fee \cdot nIssueActions \;- \\
```

Comment:
why `-` ? 
need to provide rationale for this.

Reply:
Based on discussion, removed the finalization fee altogether, Also completed the Rationale section